skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,137 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+This module is designed to facilitate image arithmetic and logical operations
+on image data sets. The included functions supplement the logical operations
+currently provided in numpy in order to provide a complete set of logical
+operations for data analysis.
+
+The new functions include:
+    logical_nand: Identifies all elements NOT included in BOTH inputs
+    logical_nor: Identifies all elements NOT included in EITHER input
+    logical_sub: Identifies all elements ONLY included in input_1
+"""
+
+import numpy as np
+from numpy import (logical_and, logical_or, logical_not, logical_xor, add,
+                   subtract, multiply, divide)
+
+def logical_nand(x1,
+                 x2):
+    """
+    This function enables the computation of the LOGICAL_NAND of two image or 
+    volume data sets. This function enables easy isolation of all data points 
+    NOT INCLUDED IN BOTH SOURCE DATA SETS. This function can be used for data 
+    comparison, material isolation, noise removal, or mask 
+    application/generation.
+
+    Parameters
+    ----------
+    x1 : {array, list, tuple, int, float}
+        Specifies the first reference data
+
+    x2 : {array, list, tuple, int, float}
+        Specifies the second reference data
+
+    Returns
+    -------
+    output : {bool, array, list, tuple}
+        Returns the resulting array to the designated variable
+
+    Example
+    -------
+    >>>input_1 = [[0,0,1,0,0], [2,1,1,1,2], [2,0,1,0,2]]
+    >>>input_2 = [[0,0,0,0,0], [2,1,1,1,2], [0,0,0,0,0]]
+    result = logical_nand(input_1, input_2)
+    >>>result
+    [out]: array([[ True,  True,  True,  True,  True],
+                  [False, False, False, False, False],
+                  [ True,  True,  True,  True,  True]], dtype=bool)
+    """
+    output = logical_not(logical_and(x1,
+                                     x2))

```

Comment:
You mean the >>> ? I saw that format in other function examples, including some of yours (I believe) and figured that it was a proper way to document usage examples in docstrings. Should these simply be removed?